images/openwrt.yaml

```diff
@@ -214,8 +214,6 @@ packages:
   - packages:
     - sudo
     action: install
-    releases:
-    - 23.05
```

Comment:
Looking at https://github.com/MusicDin/lxd-ci/actions/runs/11941866814, do I understand it right that the `manager: apk` bit is totally ignored and whatever is the package manager is used to install `sudo` as it's present in both `23.05` and `snapshot`?

Reply:
No, the used package manager in snapshot is `apk` in `23.05` is `opkg`. In the first case, the package manager from config is used, in the latter from the lxd-imagebuilder flag.

Reply:
It is not ignored. We need to tell lxd-imagebuilder which manager to use, as it wraps install/update/remove commands for different managers.